commands/command.go

```diff
@@ -262,3 +269,67 @@ func checkArgValue(v string, found bool, def Argument) error {
 func ClientError(msg string) error {
 	return &Error{Code: ErrClient, Message: msg}
 }
+
+func ExternalBinary() *Command {
```

Comment:
this whole function is very ipfs-specific. it does not belong in the commands lib